denys/Lesson01-Favorite-Toys/T01.01-Exercise-CreateLayout/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
```

Comment:
This is correct. We should use the `gradlew` wrapper instead of the system wide `gradle`.